packages/hash/frontend/src/shared/ui/link.tsx

```diff
@@ -10,10 +10,10 @@ import { styled } from "@mui/material/styles";
 import { Button } from "./button";
 import { FRONTEND_URL } from "../../lib/config";
 
+// @todo: update the regex to check against the domain of the hosted version of HASH
 export const isHrefExternal = (href: string | UrlObject) =>
   typeof href === "string" &&
-  (href === "/discord" ||
-    !/^(mailto:|#|\/|https:\/\/blockprotocol\.org)/.test(href)) &&
+  (href === "/discord" || !/^(mailto:|#|\/)/.test(href)) &&
```

Comment:
If I understand correctly, BP links are external now (i.e. open in a new tab). That makes sense, but I don't quite remember the reason for making them ‘internal’ in the first place.

Reply:
When building out the base components, we copied over some base code from the BP site and forgot to update this to track blockprotocol.org as an external link.